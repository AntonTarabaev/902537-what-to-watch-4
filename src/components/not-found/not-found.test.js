import {Router} from "react-router-dom";
import history from "@root/history";
import NotFound from "@components/not-found/not-found";

it(`Loader component rendered correctly`, () => {
  const tree = renderer.create(
      <Router history={history}>
        <NotFound/>
      </Router>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
