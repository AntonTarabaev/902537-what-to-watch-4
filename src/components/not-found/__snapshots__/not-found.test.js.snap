// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component rendered correctly 1`] = `
<div
  className="user-page"
>
  <header
    className="page-header user-page__head"
  >
    <div
      className="logo"
      style={
        Object {
          "order": "-1",
        }
      }
    >
      <a
        className="logo__link"
        href="/"
        onClick={[Function]}
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <h1
      className="visually-hidden"
    >
      WTW
    </h1>
    <h2
      className="page-title user-page__title"
    >
      404 - PAGE NOT FOUND
      <br />
      The page you are looking for doesn't exist or an other error occurred. Please go back to the home page.
    </h2>
  </header>
  <footer
    className="page-footer"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link logo__link--light"
        href="/"
        onClick={[Function]}
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="copyright"
    >
      <p>
        Â© 2019 What to watch Ltd.
      </p>
    </div>
  </footer>
</div>
`;
